tests/client/amo/components/TestAddonDetail.js

```diff
@@ -313,6 +317,34 @@ describe('AddonDetail', () => {
       rootNode.querySelector('.InstallButton-switch input').disabled);
   });
 
+  it('disables install switch and uses button for OpenSearch plugins', () => {
+    const AddSearchProviderSpy = sinon.stub(window.external,
```

Comment:
`window.external` is a non-standard property so I'd rather see a fake `_window` property passed into the component.